### DISCUSSION TITLE
OCIO and the display and channel dropdowns

### DISCUSSION THREAD

**Discussion by BrianHanke** [AcademySoftwareFoundation/OpenRV#391] [Q&A] [answered]
Currently, enabling OCIO depopulates both the channel and display dropdowns in the top toolbar. Is this something that can be controlled via _ocio_source_setup.py_? It would be great if the display dropdown changed to show all the OCIO display transforms, for example. I'm happy to hack away at it, but first I wanted to find out if _ocio_source_setup_ is the place to start or if editing other files will be required. Thanks!

**Comment by bernie-laberge** [comment]
Hello Brian,

The OCIO/Displays' RV menu should be listing all the OCIO displays from your OCIO config:
<img width="989" alt="ocio_display_menu" src="https://github.com/AcademySoftwareFoundation/OpenRV/assets/117092886/67f20d4f-a66b-485d-879a-3b95091a4b9b">

Now you might be looking for something else altogether perhaps. Please let me know if this is the case, I would be curious to see if anyone in the community had added any customization to RV relative to OCIO displays.

**Comment by BrianHanke** [comment]
@bernie-laberge Yep, that menu works great! I was thinking of the Color Channel dropdown/menu. When OCIO is active you can't isolate R/G/B etc that way.

![rv01](https://github.com/AcademySoftwareFoundation/OpenRV/assets/59420805/03349c57-0322-4ea4-b72b-f23095d856e0)

Separately, I was thinking that it would be cool if the Display dropdown got populated with all the OCIO displays. It just gets emptied out now. 

![rv02](https://github.com/AcademySoftwareFoundation/OpenRV/assets/59420805/63ab41c0-1454-4f86-bc7f-184cd7fe9850)

Would these be possible to script using _ocio_source_setup_, or would it require getting more into the guts of RV? Thanks!

**Reply by markreidvfx** [reply, marked as answer]
@BrianHanke we added support for this in
https://github.com/AcademySoftwareFoundation/OpenRV/pull/838

Also added support for dithering

**Reply by BrianHanke** [reply]
@markreidvfx Nice! Thanks for letting me know.
